python/utils/EOSEventsWithDownload.py

```diff
@@ -0,0 +1,162 @@
+from DataFormats.FWLite import Events as FWLiteEvents
```

Comment:
why this piece of code went into utils and you don't fetch it from standard heppy/cmg?

Reply:
I alternated this code by changing the print messages with the prefix "[FileFetcher]", so it's easier to detect in the logfiles which outputs are from him and whether he's responsible for the bugs. Since I wasn't sure how many other changes I will add on top of this, I copied the file and didn't want to contaminate the original one.

Reply:
ok